WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -3056,59 +3021,23 @@ function WeakAuras.ReloadTriggerOptions(data)
       WeakAuras.ReloadTriggerOptions(data);
     end
 
-    removeFuncs(displayOptions[id]);
```

Comment:
All code paths, whether for buff or generic trigger functions called this a second time after adding the trigger options, so just do it once after adding the trigger options